tests/integration/census/test_census_household_submission_data.py

```diff
@@ -1118,11 +1101,12 @@ def complete_individual_section_person_2(self):
         # Completed Individual section
         self.assertInPage('There are no more questions for Anjali Yo')
         self.post(action='save_continue')
-        self.assertInPage('The next section covers questions for Visitor 1')
+        self.assertInPage('Visitors')
```

Comment:
This is probably a bad assert as this value on in the navigation?

Reply:
Agreed will change